desktop/public_desktop/preload.js

```diff
@@ -43,11 +43,24 @@ const receiveChannels = [
 	'toggle-notification-bar',
 ];
 
+function fflagOverrides() {
+	const payload = {};
+	let fflags = process.env.WP_DESKTOP_DEBUG_FEATURES;
```

Comment:
A little style difference, I would have likely written this as 
```
	const fflags = process.env.WP_DESKTOP_DEBUG_FEATURES
		? process.env.WP_DESKTOP_DEBUG_FEATURES.split( ',' )
		: [];
	for ( let i = 0; i < fflags.length; i++ ) {
		payload[ fflags[ i ] ] = true;
	}
```
I don't think it really makes a big difference, but it allows `fflags` to be a `const` and reads a little cleaner to me.